### ISSUE TITLE
v4.4.1: GUI docker image fails due to outdated packages

### ISSUE THREAD

**Issue by sharkovsky** [fedbiomed/fedbiomed#821]
After building the GUI image, it fails to run and outputs the following error:
```bash
#0 250.6 error lru-cache@10.0.0: The engine "node" is incompatible with this module. Expected version "14 || >=16.14". Got "16.13.1"
#0 250.7 error Found incompatible module.
```

**Comment by srcansiz** [comment]
Hi @sharkovsky, 

Thanks for reporting the issue. I think building GUI interface in the docker image will always raise such problems. Instead of solving this outdated package error. I would suggest to distribute GUI build directory. It will reduce build time on docker side and provide more robust way to deploy GUI interface without depending the operating system or the containerization.  

We can create `fedbiomed/gui/ui/public` or `dist`  directory to keep complied UI files. During the deployment docker containers can directly use those files instead of building React app.

**Comment by sharkovsky** [comment]
Solved by #822
